core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -233,6 +233,34 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("addFile when file path contains spaces with recursive works") {
```

Comment:
Since this is a bug fix PR, shall we update like the following? (https://spark.apache.org/contributing.html)
```
- test("addFile when file path contains spaces with recursive works") {
+ test("SPARK-30126: addFile when file path contains spaces with recursive works") {
```

Reply:
ok , i will update as per your suggestion.